examples/react-native/src/features/InAppMessaging/Demo/Example.tsx

```diff
@@ -6,9 +6,16 @@ import { SafeAreaView, StyleSheet, Text, View } from 'react-native';
 import { withInAppMessaging } from '@aws-amplify/ui-react-native';
 import { Button, Checkbox, Radio, RadioGroup } from '../../../ui';
 import { useInAppDemo, ACTIONS, LAYOUTS, ORIENTATIONS } from './utils';
-import config from './aws-exports';
+import { VERSION } from '@env';
 
-Amplify.configure(config);
+const AMPLIFY_CONFIG_PATH =
```

Comment:
Using `src/amplifyconfiguration` - the json equivalent  of `src/aws-exports` because `require` will return the exact config object when loading a json file versus putting it behind a `default` key when loading a js file